### PR TITLE
feat: update wallet when status is changed to succeeded from API

### PR DESCRIPTION
**Pull request by lovrocolic**
## Context

Update wallet when invoice status changed to succeeded from the API

### CODE REVIEW

File: app/jobs/prepaid_credit_job.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class PrepaidCreditJob < ApplicationJob
```
Comment (jdenquin) [inline app/jobs/prepaid_credit_job.rb:3]: can you namespace it? `Wallets::PrepaidCreditJob` for example?

**Review by jdenquin — CHANGES_REQUESTED** [review]

File: app/jobs/prepaid_credit_job.rb (hunk 1 above)
Comment (lovrocolic) [reply app/jobs/prepaid_credit_job.rb]: as discussed offline, namespaced it with `Invoices`

File: app/jobs/invoices/prepaid_credit_job.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Invoices
+  class PrepaidCreditJob < ApplicationJob
```
Comment (vincent-pochet) [inline app/jobs/invoices/prepaid_credit_job.rb:4]: Looks good!

My only comment would be to suggest to move this business logic into a dedicated service with a name clearer than the job name. Right now, it's not easy to guess what the job is doing by reading its name.

**Review by vincent-pochet — APPROVED** [review]

File: app/jobs/invoices/prepaid_credit_job.rb (hunk 2 above)
Comment (lovrocolic) [reply app/jobs/invoices/prepaid_credit_job.rb:4]: seems reasonable. I also wasn't 100% happy with the naming but in the same job we handle `WalletTransaction` and `Wallet` so I wasn't sure how to name it...
Moved the logic into new service and I have named it `Wallets::ApplyPaidCreditsService`

**Pull request merged**